### ISSUE TITLE
Add support for default KV namespaces

### ISSUE THREAD

**Issue by gabbifish** [cloudflare/wrangler#502]
This is blocked on #220.

We need to do two things:
1. Make the `namespace-id` argument optional (currently it remains required).
2. Implement the logic for pulling a KV namespace from the default environment.

**Comment by ashleymichal** [comment]
closing in favor of #524 , which will address this issue.
